src/index.css

```diff
@@ -0,0 +1,131 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #fafafa;
+}
+
+.header {
+  width: 100%;
+  padding: 20px 100px;
+  border-bottom: 1px solid rgb(197, 197, 197);
+  background-color: #fff;
+  display: flex;
+}
+
+.header h1 {
+  color: #0290ff;
+}
+
+.header ul {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 30px;
+  padding: 10px 20px;
+  width: 180px;
+  list-style-type: none;
+}
+
+.links {
+  color: #5e5e5e;
+  text-decoration: none;
+}
+
+.header ul li:hover {
+  cursor: pointer;
+}
+
+.header img {
+  width: 30px;
+  height: 30px;
+  margin-left: 650px;
+}
+
+.books-list {
+  width: 100%;
+  padding: 20px 100px;
+  background-color: #fafafa;
+}
+
+.singlBook {
+  display: flex;
+  justify-content: space-between;
+  margin: 15px 0;
+  padding: 20px;
+  background-color: #fff;
+  border-radius: 10px;
+}
+
```

Comment:
### Your Design
![your-design](https://user-images.githubusercontent.com/93039359/187049125-daf0ac1f-5d50-4f17-8250-857c5447b8b7.png)

### Template
![design](https://user-images.githubusercontent.com/93039359/187049133-997f40fd-b8fd-4348-a24c-595997173205.png)

### Circlular Progressbar
![Screenshot from 2022-08-28 02-44-09](https://user-images.githubusercontent.com/93039359/187049079-d7b4a509-c7b0-4f5b-b770-bf22341dc997.png)

- [ ] Hey you have made a very good design but i guess you forgot to add the circular progressbar. Please add it to make your design look professional :blush: